### PR TITLE
Sp/groq

### PR DESCRIPTION
**Pull request by pagel-s**
Added dropdown menu to select the model to be prompted. Also added Groq support

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ dependencies:
 - arxiv
 - wikipedia
 - langchain-openai
+- langchain_groq
```
Comment (jan-janssen) [inline environment.yml:10]:
```suggestion
- langchain-groq
```

Comment (jan-janssen) [reply environment.yml:10]: The package is available https://anaconda.org/conda-forge/langchain-groq but the download does not work yet as it was just merged 5 hours ago https://github.com/conda-forge/staged-recipes/pull/26295

Comment (pagel-s) [reply environment.yml:10]: Did not saw that you just added it yesterday!
Could add it as a pip dependency, or are we strictly sticking to conda?

Comment (jan-janssen) [reply environment.yml]: Done

**Pull request merged**